examples/titanic/README.md

```diff
@@ -13,6 +13,11 @@ In order to run this example, you'll need to:
 * [install the `substra` cli](../../README.md#install)
 * [install the `substratools` library](https://github.com/substrafoundation/substra-tools)
 * [pull the `substra-tools` docker images](https://github.com/substrafoundation/substra-tools#pull-from-private-docker-registry)
+* have one or two profiles:
```

Comment:
I think we could improve the README by:
- giving more details regarding this step
- setting the URL from the skaffold setup (instead of docker-compose)

So something like the following lines could be better:

create the substra profile to define the substra network to target, for instance:
```sh
substra config --profile node-1 --username node-1 --password 'p@$swr0d44' http://substra-backend.node-1.com
```